Made games alphabetical

jacobmix opened the pull request:
Added Crash Bandicoot 4: It's About Time, and made everything alphabetical by their game name.

jacobmix committed:
Added Crash 4, and made everything alphabetical (by name)

Added Crash Bandicoot 4: It's About Time, and made everything alphabetical by their game name.

jacobmix committed:
Made games in the "Howto Use" section alphabetical, and fixed typo

Changed games in the "Howto Use" section to be alphabetical.
Fixed the type from "Call of Duty: Back Ops Cold War" to "Call of Duty: Black Ops Cold War"

jacobmix committed:
Fixed typo

Fixed "Call of Duty: Back Ops Cold War" to "Call of Duty: Black Ops Cold War"

dafzor commented:
games.ini order was more based of client used. That said, Crash Bandicoot is not included on readme.

jacobmix commented:
Mainly just wanted to makes it easier for the community to add games to games.ini themselves.
Easier to sort through it since it's alphabetical. I just added crash since i owned it.
Didn't add Crash 4 to the readme tho as i assume that's just a minimal example. Doesn't include all games or ptr versions.
Don't know if this changes the games.ini.example file tho. But i assume it just copies it over.

dafzor commented:
The readme is actually the full "official" supported list of games.

While I've expanded the scope of bnetlauncher with epic client support it hasn't been mentioned beyond advanced configuration with the bundled examples simply being the games I play.

jacobmix committed:
Added Crash Bandicoot 4 to readme

Added Crash Bandicoot 4 to readme

jacobmix commented:
Alright. Added Crash Bandicoot 4 to the readme.
May be a good idea to also sort by launcher.
I'll probably edit the gamesdb.ini later tonight or tomorrow for that.

jacobmix committed:
Sort Battlenet tittles first

Changed so Battlenet tittles are listed before the rest. PTR, and such are still listed right after the normal game.

dafzor commented on README.md:
```diff
@@ -26,22 +26,23 @@ the game starts, otherwise it will be left running.
+|codbocw        | Call of Duty: Black Ops Cold War                      |
+|codmw2019      | Call of Duty: Modern Warfare (2019)                   |
+|codmw2crm      | Call of Duty: Modern Warfare 2 Campaign Remastered    |
+|wlby           | Crash Bandicoot 4: It's About Time                    |
```
Should be cb4 not wlby

jacobmix committed:
Fixed Crash 4

Fixed Crash 4 to cb4
